plugins/include/float.inc

```diff
@@ -21,73 +24,250 @@ enum floatround_method {
 	floatround_tozero
 };
 
+/* Different units of measurement for angles */
```

Comment:
You need to keep the same following comment format here and at the others places. For consistency and for the doc parser as well.

```
/**
 *
 */
```